packages/core/state/interaction/logics.ts

```diff
@@ -35,6 +35,9 @@ import {
     COPY_FILES,
     EDIT_FILES,
     EditFilesAction,
+    editFiles,
```

Comment:
this bit around editFiles seems inconsistent; the pattern is FooAction, FOO_ACTION and for edit there is EDIT_FILES, EditFileAction, editFiles....  I guess OPEN_WITH does that too.

Reply:
Our action/logic imports are definitely somewhat ad-hoc and random at the moment. The ALL_CAPS import is a constant/type, EditFilesAction is an interface, and editFiles is the function itself. I did a slight reorganization in [06b5c60](https://github.com/AllenInstitute/biofile-finder/pull/542/commits/06b5c60796004257e732aa944bacf4f4705a6931) so that it follows a TINY bit more of a pattern